### PR TITLE
Hw25 custom map

### PR DESCRIPTION
**Pull request by ThorSilverRanger**

### CODE REVIEW

File: hw25_custom_map/js/custom_map.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+}
+
+Array.prototype.customMap = customMap;
+    let arr = [1,2,3,4,5,6,7,8,9,10], 
+    foo = function(map) {
+        return map * 10
+};
+    let myArr = arr.customMap(foo)
+
+alert(myArr);
```
Comment (vadkuze) [inline hw25_custom_map/js/custom_map.js:16]: Oh, no. Please keep in the style

File: hw25_custom_map/js/custom_map.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+function customMap(callback){
+    let arr = [];
+    for (let i=0; i<this.length; i++)  {
+        arr.push(callback(this[i]))
```
Comment (vadkuze) [inline hw25_custom_map/js/custom_map.js:4]: The default `map` has more than one argument ) Please supplement the line with `;`

**Review by vadkuze — CHANGES_REQUESTED** [review]

File: hw25_custom_map/js/custom_map.js (hunk 2 above)
Comment (ThorSilverRanger) [reply hw25_custom_map/js/custom_map.js]: supplement the line with arguments

File: hw25_custom_map/js/custom_map.js (hunk 1 above)
Comment (ThorSilverRanger) [reply hw25_custom_map/js/custom_map.js]: tried to fix the style

**Review by vadkuze — APPROVED** [review]

**Pull request merged**